src/singlyLinkedList/node.ts

```diff
@@ -0,0 +1,32 @@
+export class SinglyLinkedNode<T> {
+  constructor(value: T) {
+    this.value = value;
+    this.next = null;
+  }
+
+  next: SinglyLinkedNode<T> | null;
+  value: T | null;
```

Comment:
Since you're using Typescript, you can add `private` scope to prevent accidentally writing directly to `next` or `value` when using a LinkedList node

Reply:
Love this! Thank you!